spec/spec_helper.rb

```diff
@@ -19,3 +19,8 @@ def tmp_path(file)
 end
 
 MATCH_REQUESTS_ON = { match_requests_on: [:method, :uri, :headers, :body] }
+
+RSpec.configure do |config|
```

Comment:
Allows filtering a test with `describe/it 'test name', focus: true do` to debug specific test(s).